jscomp/syntax/tests/printer/expr/jsx.res

```diff
@@ -6,17 +6,15 @@ let x = <Foo.bar className="container" />
 let x = <Foo.baz
  className="multiline"
 />
-let x = <\"custom-tag" className="container" />
-let x = <Foo.\"custom-tag" className="container" />
```

Comment:
These tests should be possible to keep around, no? This case feels like it shouldn't be affected by the current changes.

Reply:
```
let x = <\"custom-tag" className="container" />
```
is still here (line below), it just doesn't need the quotes anymore.
I removed `<Foo.custom-tag` because I wasn't even sure of its semantics to be honest, but since we're talking about syntax, I put it back

Reply:
Ah yes, it was the Foo. one I was referring to. Perfect!